### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by nvdtf**
Closes: #782 

## Description

- Updated examples to Cadence 1.0
- Fixed bugs in some examples
- Verified all of them run

### CODE REVIEW

File: access/http/handler.go [hunk 1]
```diff
@@ -408,7 +408,7 @@ func (h *httpHandler) getExecutionResults(
 	u := h.mustBuildURL("/execution_results", opts...)
 
 	q := u.Query()
-	q.Add("block_ids", strings.Join(blockIDs, ","))
+	q.Add("block_id", strings.Join(blockIDs, ","))
```
Comment (nvdtf) [inline access/http/handler.go:411]: The endpoint was wrong in the SDK

File: examples/send_transactions/main.go [hunk 2]
```diff
@@ -40,14 +40,14 @@ func demo() {
...
-
-  prepare(acc: &Account) {}
-
-  execute {
-    log("test")
-  }
-}`)).
+		SetScript([]byte(`
+			transaction {
+				prepare(signer: auth(Storage) &Account) {}
```
Comment (nvdtf) [inline examples/send_transactions/main.go:45]: not sure how to specify `auth(None)`

**Review by bthaile — APPROVED** [review]

**Review by gregsantos — APPROVED** [review]

File: examples/send_transactions/main.go (hunk 2 above)
Comment (turbolent) [reply examples/send_transactions/main.go:45]: Simply remove the authorization
```suggestion
				prepare(signer: &Account) {}
```

Comment (nvdtf) [reply examples/send_transactions/main.go:45]: oh yeah! thanks

**Pull request merged**